### ISSUE TITLE
激活函数

### ISSUE THREAD

**Issue by huuuuusy** [huuuuusy/Deep-Learning-Practice-Everyday#1]

**Comment by huuuuusy** [comment]
# Sigmoid

![image](https://user-images.githubusercontent.com/32117906/63733724-214aef80-c8ac-11e9-8ac7-dd90ab7aa387.png)

**Comment by huuuuusy** [comment]
# Tanh

- 函数形式：
![image](https://user-images.githubusercontent.com/32117906/63734383-a2a38180-c8ae-11e9-9305-4b6f8190a9c8.png)
- 函数输出会落在(-1,1)区间

**Comment by huuuuusy** [comment]
# ReLU

- 函数形式：
![image](https://user-images.githubusercontent.com/32117906/63734620-8d7b2280-c8af-11e9-8864-de315da006ae.png)
- 特点：
    - 对正向信号重视，忽视负向信号
    - 运算简单，提升机器运行效率
- TF指令：
    - tf.nn.relu(features, name=None)
- 注意：

**Comment by huuuuusy** [comment]
# 小结

- **Tanh**函数在特征相差明显时效果较好，在循环过程中会不断扩大特征效果并且显示出来。
- 当计算的特征间的相差虽然复杂但是没有明显区别时，或者特征之间的相差不是特别大时，需要更细微的分类判别，此时**Sigmoid**效果更好。
- **ReLU**的优势是经过其处理后的数据有更好的稀疏性，即，将数据转化为只有最大的数值，其他均为0;这种变换可以最大程度保留数据特征，用大部分元素均为0的稀疏矩阵来实现。以稀疏矩阵来表示原有数据特征的方法，使得神经网络在迭代时能够取得又快又好的效果。

**Comment by huuuuusy** [comment]
# Softmax

- 函数形式：
![image](https://user-images.githubusercontent.com/32117906/63753508-cb933900-c8e5-11e9-86fd-292d12f61421.png)
- 图像：
![image](https://user-images.githubusercontent.com/32117906/63753577-e9f93480-c8e5-11e9-8719-0cd24cd95a6e.png)
- 特点：
    - Softmax用于多分类任务，它将多个神经元的输出映射到(0,1)区间内，数值越大则表示属于该分类的可能性越大。
## 参考
[【知乎】详解softmax函数以及相关求导过程](https://zhuanlan.zhihu.com/p/25723112)
